demos/https/iot_demo_https_s3_download_sync.c

```diff
@@ -261,6 +261,8 @@ int RunHttpsSyncDownloadDemo( bool awsIotMqttMode,
     char rangeValueStr[ RANGE_VALUE_MAX_LENGTH ] = { 0 };
     /* The current attempt in the number of connection tries. */
     uint32_t connAttempt = 0;
+    /* Buffer to read the Connection header value into.  The possible values are "close" and "keep-alive". */
+    char connectionValueStr[ RANGE_VALUE_MAX_LENGTH ] = { 0 };
```

Comment:
Should this be `connectionValueStr[CONNECTION_HEADER_FILED_LENGTH]`?

Reply:
I use this length because it exceeds close and keep-alive and did not want to redefine another marco. I can carry that comment from the _async.c over.

Reply:
I will just ahead and update this the size of keep-alive.